cognite_toolkit/_cdf_tk/commands/deploy.py

```diff
@@ -227,7 +227,7 @@ def sort_key(p: Path) -> int:
         if not loaded_resources:
             return ResourceDeployResult(name=loader.display_name)
 
-        capabilities = loader.get_required_capability(loaded_resources)
+        capabilities = loader.get_required_capability(loaded_resources, dry_run)
```

Comment:
Suggest specifying the keyword argument to increase readability
```suggestion
        capabilities = loader.get_required_capability(loaded_resources, read_only=dry_run)
```